src/renderer/views/Post/Post.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <div v-if="!isInvidiousAllowed">
+      {{ $t('Channel.Community.Viewing Posts Only Supported By Invidious') }}
+    </div>
+    <FtLoader v-else-if="isLoading" />
+    <template
+      v-else
+    >
+      <FtCard>
+        <FtCommunityPost
+          :data="post"
+          :single-post="true"
+          appearance="result"
+        />
+      </FtCard>
+      <FtCard>
+        <WatchVideoComments
+          :id="post.postId"
+          :channel-name="post.author"
+          :post-author-id="authorId"
+          :video-player-ready="false"
+          :force-state="null"
+          :is-post-comments="true"
+          :channel-thumbnail="post.authorThumbnails[0].url"
+          :show-sort-by="false"
+        />
+      </FtCard>
```

Comment:
As WatchVideoComments already has it's own FtCard, it displays in two cards one inside of the other, which looks a bit weird:
![comments-double-card-with-shadow](https://github.com/user-attachments/assets/d6ba3c6d-3909-4cdb-8879-8baec4608adb)